packages/shared/styles/components/organisms/SfStoreLocator.scss

```diff
@@ -12,28 +12,48 @@
     color: var(--c-dark-variant);
     @include font(
       --store-item-distance-font,
-      var(--font-light),
+      var(--font-normal),
```

Comment:
Would be good to update branch with develop and change all on the current vars, wdyt? :)

Reply:
Of course, you are right. I fixed that.